postcss.config.js

```diff
@@ -1,6 +1,29 @@
 module.exports = {
-  plugins: {
-    tailwindcss: {},
-    autoprefixer: {},
-  },
+  plugins: [
+    'tailwindcss',
+    [
+      /**
+       * prefixing all DSFR base styles to prevent CSS conflicts with parts of the website
+       * that do not use the DSFR.
+       *
+       * This is done because I couldn't manage to load the DSFR global styles only for a specific page.
+       * If you go on a LandingPage (that has the DSFR), then on the homepage (that doesn't have the DSFR),
+       * the DSFR global styles are still loaded and conflict with homepage styles without a prefix.
+       */
+      'postcss-prefix-selector',
+      {
+        prefix: ':where(.enable-dsfr)',
+        transform(prefix, selector, prefixedSelector, filePath) {
+          if (filePath.includes('/dsfr/')) {
+            if (selector === 'body' || selector === ':root') {
+              return prefix
+            }
+            return prefixedSelector
+          }
+          return selector
+        },
+      },
```

Comment:
Yeah… the problem is we can't really load precisely global CSS how we want global with Next… this is a trick to go around this barrier and prevent CSS collapsing issues between our own CSS and DSFR CSS. I needed to update this actually, it's gonna be even more weird because I stumbled upon weird bugs… stay tuned.